listenbrainz/webserver/views/playlist_api.py

```diff
@@ -143,11 +143,19 @@ def serialize_xspf(playlist: Playlist):
             playlist_extension_collaborator = ET.SubElement(playlist_extension_collaborators, "collaborator")
             playlist_extension_collaborator.text = collaborator
 
-    if playlist.additional_metadata:
-        playlist_extension_additional_metadata = ET.SubElement(playlist_extension, "additional_metadata")
-        for meta_key, meta_value in playlist.additional_metadata:
-            additional_metadata_item = ET.SubElement(playlist_extension_additional_metadata, "item", key=meta_key)
-            additional_metadata_item.text = meta_value
+        if playlist.additional_metadata:
```

Comment:
While testing the PR locally, the additional_metadata block wasn't showing up. Then I saw this entire block is over indented. It should be on the same indentation as other if blocks.